tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-perf-test-ci-pipeline.yml

```diff
@@ -22,17 +22,37 @@ jobs:
         --config RelWithDebInfo
         --enable_training
         --update --build
+        --build_wheel
         "
+        -u
```

Comment:
```-u``` is indicative of ```ORTModule``` build which installs the latest version of transformers library and (dependent on folder ```ortmodule``` being there).
I think master does not have all those changes, so this argument here is not necessary.

Reply:
Nevermind, I was mistaken. Did not see the test being run was an ortmodule test.